spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -411,14 +413,16 @@ def step(  # noqa
         self.stopwatch.record("get_observations")
 
         self.num_steps += 1
-        print(
+        self.say(
```

Comment:
There is no need for this change. Let's keep it `print`. No need to emit this as audio-out.